modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,91 @@
+/ Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a flow collector is deployed, new dashboards are available in the Console.
```

Comment:
Is it always in Grafana Loki? cc @jotak @mariomac @jpinsonneau

Reply:
Yes there is no other option so far